src/testing/slim_check/functions.lean

```diff
@@ -0,0 +1,300 @@
+
```

Comment:
This file needs a fair bit of work still:

* copyright notice
* module docs
* many definitions belong in other files